packages/contracts/deploy-config/sepolia.json

```diff
@@ -14,6 +14,21 @@
   "validatorPoolRequiredBondAmount": "0x16345785d8a0000",
   "validatorPoolMaxUnbond": 10,
   "validatorPoolRoundDuration": 300,
+  "validatorPoolTerminateOutputIndex": "0x1F9A5",
+  "validatorManagerTrustedValidator": "0xc2da150ecfaa2e275577203bb177dd4de4a2536e",
+  "validatorManagerMinRegisterAmount": "0x56BC75E2D63100000",
```

Comment:
For other reviewes, it is 100 $KRO

Reply:
Yes, I scaled down the KRO related configs comparing to the actual configs. Since the total supply is limited, so just scaled down in testnet. It's the same configs with Holesky 1.